test/brsTypes/components/RoLongInteger.test.js

```diff
@@ -0,0 +1,81 @@
+const brs = require("brs");
+const { roLongInteger, Int64, BrsBoolean, BrsString, Callable } = brs.types;
+const { Interpreter } = require("../../../lib/interpreter");
+
+describe("roLongInteger", () => {
+    describe("equality", () => {
+        it("compares to intrinsic Int64", () => {
+            let a = new roLongInteger(new Int64(10));
```

Comment:
Can you actually use long integers here? I think javascript should be able to handle integers greater than 32 bits

Reply:
It is done in the same way as in [RoDouble.test.js](https://github.com/sjbarag/brs/blob/c8067f2c9f11a6de969a744ab26cacec37a97a4e/test/brsTypes/components/RoDouble.test.js#L8). Can you please add more context to what you are expecting here?

Reply:
done